odoo/models.py

```diff
@@ -2488,7 +2488,7 @@ def _auto_init(self):
             if fields_to_compute:
                 @self.pool.post_init
                 def mark_fields_to_compute():
-                    recs = self.with_context(active_test=False).search([])
+                    recs = self.with_context(active_test=False).search([], order='id')
```

Comment:
:+1:
While you're modifying this part, could you also:
1) avoid the logs and the `add_to_compute` when recs is empty ?
2) add the model name in the logs, for some module installation, we get two identical logs, because two fields from different models have the same name.
(I had those changes in another PR, but you're too busy to review it :D).